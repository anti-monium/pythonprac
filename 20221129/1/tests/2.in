class C(metaclass=dump):
	def __init__(self, val1, val2):
		self.val = val1 ** val2

	def kwd(self, start, stop, step=1):
		return [self.val / i for i in range(start, stop, step)]

c = C(1, 3)
print(*c.kwd(1, 10))
d = C(2, 10)
print(*d.kwd(2, 16, step=3))
