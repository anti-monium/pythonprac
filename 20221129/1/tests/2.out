__init__: (1, 3), {}
kwd: (1, 10), {}
1.0 0.5 0.3333333333333333 0.25 0.2 0.16666666666666666 0.14285714285714285 0.125 0.1111111111111111
__init__: (2, 10), {}
kwd: (2, 16), {'step': 3}
512.0 204.8 128.0 93.0909090909091 73.14285714285714
